scripts/update-integrations.mjs

```diff
@@ -106,11 +112,7 @@ async function unsafeUpdateAllIntegrations() {
 		[...packagesMap.keys(), ...allowlist].filter((pkg) => !blocklist.includes(pkg)),
 	);
 
-	const pathname = path.resolve(
-		path.dirname(fileURLToPath(import.meta.url)),
-		"../src/content/integrations/*.md",
```

Comment:
This broke on Windows because `path.resolve` returns backslashes, which `tiny-glob` doesn't accept. Globs aren't meant to be manipulated like paths anyway; they're meant to be handwritten, which is why I inlined the string and passed a `cwd` instead